kalitectl.py

```diff
@@ -571,6 +572,27 @@ def start(debug=False, watch=False, daemonize=True, args=[], skip_job_scheduler=
             'server.socket_port': port
         })
 
+    # idealy we'd have a setting element, but these are imported much later
+    # and it will require a much deeper change
+    idle_timeout = int(os.getenv("KALITE_IDLE_TIMEOUT", 0))
```

Comment:
I still don't see the problem and I think it's more readable to be explicit about the condition instead of relying on 0 being false, but I don't feel as strong as to keep pushing, so merging